Appraisals

```diff
@@ -1,7 +1,32 @@
 appraise "sass-3-4" do
   gem "sass", "3.4.0"
+
+  # using rails 4 as a base for this appraisal
+  gem "rails", "~> 4.2"
+  gem "rspec-rails", "~> 3.4.2"
 end
 
 appraise "bourbon-5" do
   gem "bourbon", "5.0.0.alpha.0"
+
+  # using rails 4 as a base for this appraisal
+  gem "rails", "~> 4.2"
+  gem "rspec-rails", "~> 3.4.2"
+end
+
+appraise "rails-4" do
+  gem "rails", "~> 4.2"
+  gem "rspec-rails", "~> 3.4.2"
+end
+
+appraise "rails-5" do
+  gem "rails", "5.0.0.rc1", "< 5.1"
+  gem "rspec-rails", "~> 3.5.0.beta1"
+  gem "kaminari", github: "amatsuda/kaminari", branch: "0-17-stable"
+
+  # this fork has rails at >= 4.2, allowing rails5 to work.
+  gem "administrate-field-image", github: "pedrocarmona/administrate-field-image"
+
+  # rails 5+ extracts this gem, which is need for shoulda-matchers:
+  gem "rails-controller-testing"
```

Comment:
Yep, just took a dive into Shoulda Matchers. Latest update was in January and there's tons of low hanging fruit when it comes to issues. I'll open a PR to remove the gem. The gem is great for making sure you didn't leave any typos around when you make a new association, but beyond that, I don't really find that much use for it.

Anywho, going to remove because the gem isn't up to date and we don't really need it. We don't use Shoulda with Solidus.